### PR TITLE
feat: add qa-data-seeder skill

### PR DESCRIPTION
**Pull request by DavertMik**
## Summary

Adds a new `qa-data-seeder` skill and bundles it into the `test-automation` plugin.

The skill prepares test data for a feature in a given environment:

- Analyzes the feature's implementation: fields, validation rules, relationships, business variations, API endpoints.
- Collects environment details: platform (web/mobile), base URL, environment kind, auth, target account.
- Proposes data categories (regular + edge cases) and asks for the exact item count before seeding.
- Enforces a balanced dataset: at least 70% regular data, at most 30% edge cases; never produces broken data — every record must pass app validation.
- Seeds via existing seed mechanisms, a seed script in the app's language, or REST API; uses faker/lorem-ipsum generators for values.
- Tags all seeded records with a run marker for verification and cleanup; generated artifacts live in gitignored `.testeiya/seed-data/`.

## Changes

- `skills/qa-data-seeder/SKILL.md` — the skill
- `plugins/test-automation/skills/qa-data-seeder` — symlink into the plugin
- `README.md` — listed in the Automation table and the test-automation plugin row

🤖 Generated with [Claude Code](https://claude.com/claude-code)

### CODE REVIEW

**Comment by olexandr13** [comment]
skill text is fine
would like to test/evaluate in on my own project and then provide feedback

File: skills/qa-data-seeder/SKILL.md [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+Analyze how a feature is implemented, design a balanced dataset that covers every category QA needs, and populate it into a target environment so the feature can be tested manually or by automation.
+
+## Critical Constraints
+
+- **Never seed production.** Confirm the target environment is a test/dev/staging one before writing any data. If in doubt, stop and ask.
+- **Never produce broken data.** Every seeded record must pass the application's own validation. Edge cases are extreme-but-valid values, not corrupt ones. Malformed input belongs in negative tests executed live, never in persisted seed data.
+- **Balance the dataset: at least 70% regular (expected) data, at most 30% edge cases.**
+- **Do not seed until the user approves the categories and the exact item count** (Step 3 gate).
+- Credentials, tokens, and connection strings come from env vars or user input — never hardcode them.
+- Prefer the project's existing seeding mechanisms (factories, seed scripts, fixtures, admin console) or REST API. Write a new script only when nothing exists; keep it small; Write seed script in language of application.
```
Comment (DavertMik) [inline skills/qa-data-seeder/SKILL.md]:
```suggestion
- Prefer the project's existing seeding mechanisms (factories, seed scripts, fixtures, admin console) or REST API or MCP. Write seed script in language of application.
```

File: skills/qa-data-seeder/SKILL.md [hunk 2]
```diff
@@ -0,0 +1,83 @@
+---
+name: qa-data-seeder
+description: Prepare and seed test data for a feature into a target environment. Analyzes the feature's implementation, proposes data categories covering regular and edge cases, and populates the environment via existing seed mechanisms, REST API, or code access. Use when the user asks to seed test data, populate a test or staging environment, prepare data for manual/QA testing of a feature, generate a dataset for a feature, or create fixtures in an environment.
```
Comment (DavertMik) [inline skills/qa-data-seeder/SKILL.md]:
```suggestion
description: Prepare and seed test data for a feature into a target environment. Analyzes the feature's implementation, proposes data categories covering regular and edge cases, and populates the environment via existing seed mechanisms code access, REST, MCP. Use when the user asks to seed test data, populate a test or staging environment, prepare data for manual/QA testing of a feature, generate a dataset for a feature, or create fixtures in an environment.
```

File: skills/qa-data-seeder/SKILL.md [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+
+Read the implementation before proposing anything. Collect:
+
+- Entities and models the feature touches, with every field: type, format, length limits, numeric ranges, enums, required vs optional, uniqueness, defaults.
+- Validation rules — application-level and DB constraints/migrations.
+- Relationships and prerequisites: which records must exist before others can be created.
+- Business variations that change behavior: roles, statuses, types, plans/tiers, locales, feature flags, time-dependent logic.
+- API endpoints (or UI flows) that create these records, and what they require.
+
+If there is no code access, fall back to the API schema (OpenAPI/Swagger, GraphQL introspection) or ask the user for docs.
```
Comment (DavertMik) [inline skills/qa-data-seeder/SKILL.md]:
```suggestion
If there is no code access, fall back to the API schema (OpenAPI/Swagger, GraphQL introspection) or ask the user for docs. If MCP available - use it.
```

File: skills/qa-data-seeder/SKILL.md [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+❓ Ask for what the code cannot tell you:
+
+- Platform: web, mobile, or API-only. 
+- Base URL if web; API base URL for seeding.
+- Environment kind (local, dev, staging) — confirm it is not production.
+- Access channel available: existing seed mechanism, REST API, DB/console access.
+- How to authenticate: which env vars hold tokens or credentials.
+- Which account/tenant/workspace to seed into, or whether to create a fresh one.
+
+Pick the seeding channel in this order: existing seed mechanism → app console/code → new small script -> REST API.
```
Comment (DavertMik) [inline skills/qa-data-seeder/SKILL.md]:
```suggestion
Pick the seeding channel in this order: existing seed mechanism → app console/code → new small script -> REST API -> MCP .
```
